cypress/e2e/user-flows/scenario_slice_association.cy.js

```diff
@@ -11,7 +11,8 @@ const sliceName = `TestSlice${random}`;
 const sliceId = `test_slice${random}`; // generated automatically from the slice name
 const sliceLib = ".--slices";
 
-describe("I am an existing SM user (Next) and I want to associate a Slice to a CT and review my experience.", () => {
+// TODO: Use this test suite for Page Types since new Custom Type don't have Slice Zone anymore
```

Comment:
Is this mentioned in a ticket somewhere so we know when to come back to it?

Reply:
https://linear.app/prismic/issue/DT-1316/aauser-i-can-edit-custom-types-with-features-limited-by-the-format

I just added it in this issue, better you're right, so we don't forget.